skills_ml/algorithms/embedding/models.py

```diff
@@ -133,3 +138,39 @@ def transform(self, X):
     def fit_transform(self, X, y=None):
         self.fit(X, y)
         return self.transform(X)
+
+
+def visualize_in_tensorboard(model, output_dirname=None, host="127.0.0.1"):
+    if output_dirname is None:
+        output_dirname =  model.model_name.split('.')[0]
+
+    meta_file = f"{output_dirname}_metadata.tsv"
+    placeholder = np.zeros((len(model.wv.index2word), model.vector_size))
```

Comment:
I tried to search for where this is used so I could understand what it did but it looks like it's unused